packages/core/src/clients/dynamodbClient.ts

```diff
@@ -0,0 +1,5 @@
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb'; // ES Modules import
```

Comment:
@vincentzan you and Anaïs are stepping on each others toes. You both have created an DDB client